src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -65,6 +69,18 @@ module sonic-acl {
 					type stypes:packet_action;
 				}
 
+				leaf MIRROR_INGRESS_ACTION {
+					type leafref {
+						path "/sms:sonic-mirror-session/sms:MIRROR_SESSION/sms:MIRROR_SESSION_LIST/sms:name";
+					}
+				}
+
+				leaf MIRROR_EGRESS_ACTION {
+					type leafref {
+						path "/sms:sonic-mirror-session/sms:MIRROR_SESSION/sms:MIRROR_SESSION_LIST/sms:name";
+					}
+				}
```

Comment:
There are INGRESS_ACTION and EGRESS_ACTION, I think we can check acl rule. For example, acl stage is ingress, MIRROR_EGRESS_ACTION is invalid.

Reply:
https://github.com/Azure/sonic-mgmt/blob/master/tests/everflow/everflow_test_utilities.py#L147-L150
Looks valid action.